img/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="img/mstile-150x150.png"/>
+            <TileColor>#00aba9</TileColor>
```

Comment:
Where does this teal color come from? It is not a color used in our interface.
In fact, the interface does not have many color so I think we should not force a color and let the browser use whatever native color it wants.

Reply:
I couldn't find a more suitable color - feel free to choose another one. I don't have a good talent for colors.

Isn't it better if we determine a suitable color ourselves than let the system choose a random one?

Reply:
Couldn't we use the green from the logo (`#119352`)?

Reply:
Yes, this should look good :slightly_smiling_face: I changed it in the PR.